environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AzureParameters.java

```diff
@@ -25,6 +25,9 @@
     @Convert(converter = ResourceGroupUsagePatternConverter.class)
     private ResourceGroupUsagePattern resourceGroupUsagePattern;
 
+    @Column(name = "encryption_parameter_keyUrl")
+    private String resourceEncryptionParameterKeyUrl;
```

Comment:
What about Vault usage? I though we agreed that encryption key URLs should be also kept there like the DH EBS encryption KMS ARNs. So this probably should be some `Secret<String>` thing or however it is done elsewhere.